docs/modules/ROOT/pages/includes/server-getting-started.adoc

```diff
@@ -34,8 +34,6 @@ WARNING: You probably already have this configuration if you created your applic
 
 Now, create the directory `openapi` under your `src/main/resources` path and add the OpenAPI spec files there. We support JSON, YAML and YML extensions.
 
-To fine tune the configuration for each spec file, add the following entry to your properties file. In this example, our spec file is in `src/main/openapi/petstore.json`:
```

Comment:
Hi @carlesarnal I remove this line, this extension does not get spec files from `src/main/openapi` dir is it true?